FreeRTOS-Plus/Source/FreeRTOS-Plus-TCP/FreeRTOS_ARP.c

```diff
@@ -219,17 +223,17 @@ uint8_t ucMinAgeFound = 0U;
 
 	#if( ipconfigARP_STORES_REMOTE_ADDRESSES == 0 )
 		/* Only process the IP address if it is on the local network.
-		Unless: when '*ipLOCAL_IP_ADDRESS_POINTER' equals zero, the IP-address
-		and netmask are still unknown. */
+		 * Unless: when '*ipLOCAL_IP_ADDRESS_POINTER' equals zero, the IP-address
+		 * and netmask are still unknown. */
 		if( ( ( ulIPAddress & xNetworkAddressing.ulNetMask ) == ( ( *ipLOCAL_IP_ADDRESS_POINTER ) & xNetworkAddressing.ulNetMask ) ) ||
-			( *ipLOCAL_IP_ADDRESS_POINTER == 0ul ) )
+			( *ipLOCAL_IP_ADDRESS_POINTER == 0uL ) )
```

Comment:
Why `uL` vs `UL`?  Let's be consistent and use the same convention everywhere, I will suggest `UL`.

Reply:
I believe pc-lint likes a small 'u'. But both lint and coverity like big 'L'. Thats why 'uL' instead of other combinations...

Reply:
Sticking with 'UL'